src/composables/aeSdk.ts

```diff
@@ -45,24 +42,28 @@ import { useModals } from './modals';
 let aeSdk: AeSdkSuperhero;
 let aeSdkBlocked = false;
 let storedNetworkName: string;
-const isAeSdkReady = ref(false);
+
 const nodeNetworkId = ref<string>();
+
+const isAeSdkReady = ref(false);
+const isAeNodeReady = ref<boolean>(false);
+const isAeNodeConnecting = ref<boolean>(false);
+const isAeNodeError = ref<boolean>(false);
```

Comment:
```suggestion
Enum ConnectionStatus = {
   Ready,
   Connecting,
   Error,
};
const aeNodeConnectionStatus = ref<ConnectionStatus>(ConnectionStatus.Connecting);
```

I remember having this discussion before. Why wouldn't this single variable wouldn't work?

Reply:
Because maybe at the beginning the amount of variables is bigger, but with one variable you'll still need to establish things like `isError`, `isConnected` etc... And those would require having computed properties. So overall we will end up with more code.